### ISSUE TITLE
Bot not claiming wished characters. Bot doesn't even look at characters anymore! NEEDS TO BE FIXED ASAP

### ISSUE THREAD

**Issue by joshoea** [chiyadev/MudaeFarm#182]
The bot should say "ignoring _____ because not on wishlist, however bot doesn't do that anymore. It simply has no message! I have claiming on too!
![Capture2222222222222222222222](https://user-images.githubusercontent.com/67028136/86197498-5f3b2480-bb0a-11ea-89ea-f69497f44f28.PNG)
This is a really urgent issue! I missed 2 wishes because of this

**Comment by phosphene47** [comment]
I'm not sure. [This issue](https://github.com/chiyadev/MudaeFarm/issues/184#issuecomment-653884428) is quite recent and it seems to say the bot is working.

**Comment by joshoea** [comment]
Let me try this again. I had to turn toggleclaimrolls off for claiming to work. With toggleclaimrolls on it didn't work for claiming before. Let me try again.

**Comment by joshoea** [comment]
No this still is a issue @phosphene47 . Sorry for @ mentioning, idk if you will get post replies after so long. It claims regularly but It doesn't say "ignoring character" as it still doesn't detect characters when $togglekakerarolls and $toggleclaimrolls are on. I assume you know what those commands do? They basically display claim ranks /kakera on rolls but that changes waifu format so bot doesn't detect it

**Comment by JohnJing731** [comment]
sooo I'm a beginner here. I configured everything right and added my channel for the bot to be in but the bot isnt in my server. And it keeps saying did not receive expected Mudae response. I need help ASAP
